lib/myfinance/resources/financial_account.rb

```diff
@@ -0,0 +1,21 @@
+module Myfinance
+  module Resources
+    class FinancialAccount < Base
+      #
+      # Create a payable/receivable account
+      #
+      # [API]
+      #   Method: <tt>POST /entities/:entity_id/payable_accounts</tt>
+      #   Method: <tt>POST /entities/:entity_id/receivable_accounts</tt>
+      #
+      #   Documentation: https://app.myfinance.com.br/docs/api/payable_accounts#post_create
+      #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts#post_create
+      #
+      def create(entity_id, params = {})
+        http.post(endpoints[:create].gsub(':entity_id', entity_id.to_s), body: { resource_key => params }) do |response|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.